toolkits/google_sheets/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "arcade_google_sheets"
-version = "2.0.0"
+version = "2.1.0"
```

Comment:
This /might/ be a breaking change (for large sheets) that previously would have returned all the cols and rows it could ahve